lib/jekyll/commands/new.rb

```diff
@@ -133,10 +133,9 @@ def bundle_install(path)
           Jekyll::External.require_with_graceful_fail "bundler"
           Jekyll.logger.info "Running bundle install in #{path.cyan}..."
           Dir.chdir(path) do
-            if ENV["CI"]
-              system("bundle", "install", "--quiet")
-            else
-              system("bundle", "install")
+            _, output = Jekyll::Utils::Exec.run("bundle", "install", "--quiet")
```

Comment:
More importantly, why do you care about the output and not the status when you tell bundler to shut up while it's doing work?